lib/CXGN/Trial/ParseUpload/Plugin/MultipleTrialDesignGeneric.pm

```diff
@@ -360,8 +363,12 @@ sub _validate_with_plugin {
     my @accessions_missing = @{$accessions_hashref->{'missing'}};
     my @multiple_synonyms = @{$accessions_hashref->{'multiple_synonyms'}};
 
-    if (scalar(@accessions_missing) > 0) {
-        push @error_messages, "Accession(s) <strong>".join(',',@accessions_missing)."</strong> are not in the database as uniquenames or synonyms.";
+    # Allow stock names to be crosses, so check missing accessions as crosses
+    my $crosses_hashref = $validator->validate($schema, 'crosses', \@accessions_missing);
```

Comment:
Sorry that I missed another point in the previous comment. For trial, we allow 3 stock types (accession, cross, family name). We have a validate plugin to validate all 3 stock types ('accessions_or_crosses_or_familynames')

Reply:
Thanks, I wasn't aware of that validation type - that makes it easy.  Family name is now included as an option.